index.html

```diff
@@ -317,7 +317,8 @@ <h3>Normative Specifications</h3>
                     <dt id="integrity" class="spec">Linked Data Integrity (LDI)</dt>
                     <dd>
                         <p>
-                            This specification defines a framework for expressing, in an RDF Graph, proofs of integrity of RDF Datasets. The group defines that framework to work with RDH, although the hashing algorithm, and other constituents of proofs of integrity (including the purpose of the proof), are identified as assertions, allowing the same framework to be used with other algorithms. Beyond the generic (normative) framework, this deliverable also provides a normative definition for Linked Data Signatures, as a prominent approach used as a proof of integrity. The specification enables a 3rd party to verify the integrity of the data.
+                            This specification defines a framework for expressing, in an RDF Graph, proofs of integrity of RDF Datasets. Since RDF follows the <a href="https://en.wikipedia.org/wiki/Open-world_assumption">Open World Assumption</a>, it is recognized that 'proof of integrity' cannot apply to arbitrarily large RDF Datasets in all contexts. Nevertheless, there are contexts where the integrity of the data <em>can</em> be assessed with confidence. <a href="https://www.w3.org/TR/vc-data-model/">Verifiable Credentials</a> are a prime example of this, where one or more assertions are made by an identified party concerning an identified subject. In contrast, asserting 'proof of integrity' over the corpus of RDF embedded in billions of Web pages would be meaningless.</p>
```

Comment:
```suggestion
                            This specification defines a framework for expressing, in an RDF Graph, proofs of integrity of RDF Datasets. Since RDF follows the <a href="https://en.wikipedia.org/wiki/Open-world_assumption">Open World Assumption</a>, it is recognized that 'proof of integrity' cannot apply to arbitrarily RDF Datasets in all contexts. Nevertheless, there are contexts where the integrity of the data <em>can</em> be assessed with confidence. <a href="https://www.w3.org/TR/vc-data-model/">Verifiable Credentials</a> are a prime example of this, where one or more assertions are made by an identified party concerning an identified subject. In contrast, asserting 'proof of integrity' over the corpus of RDF embedded in billions of Web pages would be meaningless.</p>
```
I don't think that the emphasis should be on the size of the graph here (especially not after invoking the open world assumption). The argument about the size is still present in the last sentence of the paragraph.

Reply:
The suggestion here just removes the word `large`, leaving the sentence grammatically incorrect. Perhaps this instead:

```suggestion
                            This specification defines a framework for expressing, in an RDF Graph, proofs of integrity of RDF Datasets. Since RDF follows the <a href="https://en.wikipedia.org/wiki/Open-world_assumption">Open World Assumption</a>, it is recognized that 'proof of integrity' is not useful (nor feasible to compute) for all RDF Datasets in all contexts. Nevertheless, there are contexts where the integrity of the data <em>can</em> be assessed with confidence. <a href="https://www.w3.org/TR/vc-data-model/">Verifiable Credentials</a> are a prime example of this, where one or more assertions are made by an identified party concerning an identified subject. In contrast, asserting 'proof of integrity' over the corpus of RDF embedded in billions of Web pages would be meaningless.</p>
```

Reply:
@pchampin,

> I don't think that the emphasis should be on the size of the graph here (especially not after invoking the open world assumption). The argument about the size is still present in the last sentence of the paragraph.

I think asserting that proofs of integrity aren't useful (nor feasible to compute) on some datasets (and certainly the size of them is a factor here) is an acceptable thing to do. Perhaps with my suggested language about utility and feasibility (vs. perhaps too vague "cannot apply") the last sentence that references size via "billions of Web pages" becomes less of an issue?

Reply:
I wonder if we can steal a page from the "same origin" playbook that browsers use. Could the majority/all of what's being signed be considered "same origin"? That doesn't mean you can't take data from different origins and evaluate it... just that you (origin) are putting together a package of data to canonicalize and sign... so it all effectively originates from you.

I realize that this wording can fall apart when you start injecting data from other sources into your dataset that are/are not signed. It's just an idea... putting it out there if others can craft it into something more useful.

Reply:
@msporny, let's not even try to go there unless the wording above is insufficient for some reason. It seems like it should address the raised concerns to me.

Reply:
Can we approve my suggestion if everyone is onboard and get this PR merged? Does this PR address the remaining concerns?

Reply:
@philarcher you have the pen on this... If you are out of time, I can try to take over (but that would mean a separate PR, I cannot edit yours).

Reply:
Actually... I believe I could "commit the suggestion" and let github do the work. But I do not want to do that without @philarcher nodding.